components/ActionBar/index.js

```diff
@@ -392,14 +390,14 @@ const ActionBar = ({
     },
     {
       title: t('article/actionbar/userprogress'),
-      element:
-        document && document.userProgress && displayMinutes > 1 ? (
-          <UserProgress
-            documentId={document.id}
-            userProgress={document.userProgress}
-          />
-        ) : null,
-      show: document && document.userProgress && displayMinutes > 1 && !podcast
+      element: (
+        <UserProgress
+          documentId={document.id}
+          userProgress={document.userProgress}
+          displayMinutes={displayMinutes}
+        />
+      ),
+      show: !!document
```

Comment:
Why is the `UserProgress` element coming up twice? (I'm sure there is a good reason, but I don't get it...)

Reply:
yes, the first instance is for       `modes: ['articleOverlay', 'feed', 'bookmark', 'seriesEpisode'],`, the second instance is for the second row of the article top. Could be refactored.